abstract_specification.md

```diff
@@ -250,117 +238,30 @@ Parameters:
-- value_filter - an optional [value filter](#value-filters) to apply to the results. Defaults to no filter. The `soma_rowid` pseudo-column can not be filtered.
-- platform_config - optional storage-engine specific configuration
-
-The `read` operation will return a language-specific iterator over one or more Arrow Table objects, allowing the incremental processing of results larger than available memory. The actual iterator used is delegated to language-specific SOMA specs.
-
-### Operation: write()
-
-Write an Arrow.RecordBatch or Arrow.Table to the persistent object. Rows already present in the object are overwritten and new rows are added.
-
-```
-write(Arrow.RecordBatch values, platform_config)
-write(Arrow.Table values, platform_config)
-```
-
-Parameters:
-
-- values - an Arrow.RecordBatch or Arrow.Table containing all columns, including `soma_rowid` and `soma_joinid`. The schema for the values must match the schema for the SOMADataFrame.
-- platform_config - optional storage-engine specific configuration
-
-The `values` parameter must contain a `soma_rowid` (`int64` in range `[0, 2^63-1)`) column, indicating which rows are being written.
-
-## SOMAIndexedDataFrame
-
 > ⚠️ **To be further specified** -- all methods need specification.
 >
 > Summary of operations:
 
 | Operation                               | Description                                                                    |
 | --------------------------------------- | ------------------------------------------------------------------------------ |
-| create(uri, ...)                        | Create a SOMAIndexedDataFrame.                                                 |
-| delete(uri)                             | Delete the SOMAIndexedDataFrame specified with the URI.                        |
-| exists(uri) -> bool                     | Return true if object exists and is a SOMAIndexedDataFrame.                    |
+| create(uri, ...)                        | Create a SOMADataFrame.                                                        |
+| delete(uri)                             | Delete the SOMADataFrame specified with the URI.                               |
+| exists(uri) -> bool                     | Return true if object exists and is a SOMADataFrame.                           |
 | get metadata                            | Access the metadata as a mutable [`SOMAMetadataMapping`](#SOMAMetadataMapping) |
-| get soma_type                           | Returns the constant "SOMAIndexedDataFrame"                                    |
+| get soma_type                           | Returns the constant "SOMADataFrame"                                           |
 | get schema -> Arrow.Schema              | Return data schema, in the form of an Arrow Schema                             |
 | get is_indexed -> bool                  | Return true.                                                                   |
```

Comment:
we don't need this method anymore - there is never a case where it is distinguishing...

Reply:
@bkmartinjr we do need it (it's for all classe); I think you're thinking of `is_indexed`

Reply:
... which I just removed on my last commit -- thanks!

Reply:
Yes, I made the comment on the `is_indexed` line - you guessed right.  Sorry for being opaque!